docs/source/advanced_usage.rst

```diff
@@ -124,7 +124,11 @@ and when calling ``validate`` for certain repo, pass the token:
 
     code_validator.validate(solution_repo=solution_repo, validator_token='min_max_challenge')
 
-The validator won't be executed for any other repository.
+If you wish to pass multiple tokens for certain repo, pass tokens as a list:
+
+    code_validator.validate(solution_repo=solution_repo, validator_token=['min_max_challenge', 'some_other_token'])
```

Comment:
It would be nice to have mutually exclusive `validator_token` (a string) and `validator_tokens` (an iterable) arguments. Raise `ValueError` if both are provided.